rsts/index.rst

```diff
@@ -67,9 +67,12 @@ Flyte Documentation
 
 Flyte is an open-source, container-native, structured programming and distributed processing platform. It enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
 
+It is implemented in Golang, and designed for container native execution and connecting external services. For external services, it offers backend plugins- allowing separating plugins from user code (Python library).
```

Comment:
Can we remove this? I'm not sure if it fits here, and I don't think we need to specify about external services.